utils/sl/env_modules.py

```diff
@@ -201,3 +162,9 @@ def load_mpi(mpi):
     if _env_module.env_module_init is None:
         _env_module.env_module_init = _env_module()
     return _env_module.env_module_init.load_mpi(mpi)
+
+def list_mpis(mpi):
+    """global function to return list of MPIs"""
+    if _env_module.env_module_init is None:
+        _env_module.env_module_init = _env_module()
+    return _env_module._mpi_map[mpi]
```

Comment:
(pylint-protected-access) Access to a protected member _mpi_map of a client class